packages/venia-ui/lib/components/AddressBookPage/addressBookPage.js

```diff
@@ -21,8 +20,13 @@ const AddressBookPage = props => {
     });
     const { customerAddresses, handleAddAddress } = talonProps;
 
+    const { formatMessage } = useIntl();
     const classes = mergeClasses(defaultClasses, props.classes);
 
+    const PAGE_TITLE = formatMessage({
+        id: 'Address Book',
+        defaultMessage: 'Address Book'
```

Comment:
Also I don't think default message is necessary when there are no variables in the string. That said, may be good practice to use?

Reply:
I think it's best practice to use in all cases. The jest mocks are also currently expecting it.

Reply:
Ok! I'm cool with that. Since we're moving to `context.id` pattern, a default message will help provide some context in the component so you don't have to look it up in the json.